docs/source/conf.py

```diff
@@ -63,3 +66,21 @@
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ["_static"]
+
+
+def build_mapping_tables(app):
+    """Build the library mappings tables."""
+    env = Environment(loader=FileSystemLoader("docs/source"))
+    template_file = env.get_template("table_template.j2")
+    for dict_name in lib_mapper._LIST_OF_MAP_DICTS:
```

Comment:
What about something like this instead of static definition? 

```
from netutils import lib_mapper
LIST_OF_MAP_DICTS = []

for key in dir(lib_mapper):
    if key.endswith("MAPPER_REVERSE") or key.endswith("_MAPPER") and not key.startswith("_"):
        LIST_OF_MAP_DICTS.append(key)
```

Reply:
Totally forgot about `dir()`. I added a few more conditionals in there to account for 2 of the mappings we don't pull into the docs. Maybe we want them in there though? It's the `MAIN` and `NETMIKO` mapping dictionaries.